curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3ef6e0a81099d9a697b550.md

```diff
@@ -43,6 +45,12 @@ Your new `a` element should link to `https://www.freecodecamp.org`. Remember tha
 assert(document.querySelector("footer > p > a")?.href === "https://www.freecodecamp.org/");
 ```
 
+Your new `a` element should have the `target` attribute set to `_blank`.
+
+```js
+assert(document.querySelector("footer > p > a")?.target === "_blank");
```

Comment:
please use `assert.isEqual`

Reply:
updated the code to use `assert.isEqual` as requested. Please review the changes. Thanks!